app/NavEntry.js

```diff
@@ -29,6 +34,27 @@ class MainNavigation extends React.Component {
     }
   }
 
+  async componentDidMount() {
+    const { t } = this.props;
+    try {
+      const { data } = await fetch(BULLETINS_URL);
+      const bulletins = data.bulletins;
+      const lastBulletin = await GetStoreData('lastBulletin', true);
+      console.log(lastBulletin);
+
+      if (bulletins[0].order != lastBulletin) {
+        console.log(bulletins[0].order);
+        PushNotification.localNotification({
+          title: t('label.new_bulletin_title'),
```

Comment:
I think new_bulletin_available_title makes more sense, please change it in every resource you created
```suggestion
          title: t('label.new_bulletin_available_title'),
```